completions/bash/podman

```diff
@@ -716,16 +716,22 @@ _podman_container_attach() {
 }
 
 _podman_container_checkpoint() {
-     local options_with_args="
-     --help -h
-     "
      local boolean_options="
-     --keep
+     -a
+     --all
+     -h
+     --help
      -k
+     --keep
+     -l
+     --latest
+     -R
+     --leave-running
+     --tcp-established
```

Comment:
This is such a long param.  Can we also use '-t' or will that get a little wonky as we use -t for other purposes in different commands?

Reply:
Yes, it is long, but I would prefer to not change it. My reasons for not changing is to keep the same parameter as CRIU and runc. Some options already differ between CRIU and runc which I think makes it more complicated than necessary. So my goal is to stay as close as possible to CRIU to avoid further confusion of the involved parameters. With TAB-completion it should be not too difficult.

There are also more TCP related parameters in CRIU (`--tcp-close` - just close all open TCP connections and `--skip-in-flight` to ignore TCP states other than established and open), so that there is at least another parameter which could be shortened to `-t`.

Having said all that, I can add `-t` if it would offer a better user interface.